olmv1.md

```diff
@@ -0,0 +1,298 @@
+# Kubectl Operator OLMv1 Commands
+
+Most of the kubectl operator plugin subcommands are aimed toward managing OLMv0 operators.
+
+[OLMv1](https://github.com/operator-framework/operator-controller/) has two main on-cluster CRDs, both of which can be managed by the kubectl operator `olmv1` subcommand. These are the [`ClusterCatalogs`](https://github.com/operator-framework/operator-controller/blob/main/api/v1/clustercatalog_types.go) and [`ClusterExtensions`](https://github.com/operator-framework/operator-controller/blob/main/api/v1/clusterextension_types.go). Within the repository, these are defined in `internal/cmd/olmv1.go`, which in turn references `internal/cmd/internal/olmv1` and `internal/pkg/v1`.
+
+```bash
+$ kubectl operator olmv1 --help
+
+Manage extensions via `olmv1` in a cluster from the command line.
+
+Usage:
+  operator olmv1 [command]
+
+Available Commands:
+  create      Create a resource
+  delete      Delete a resource
+  get         Display one or many resource(s)
```

Comment:
I think the most idiomatic phrasing here would be "Display one or more resources" or perhaps just "Display resources" which captures both cases

Reply:
Keeping the comment unresolved since follow up work on improving help text https://github.com/operator-framework/kubectl-operator/issues/237 needs to address this.